### PR TITLE
MOTECH-2578: Changed serviceInterface field in DB from openmrs19 to openmrs.

### PR DESCRIPTION
**Pull request by pkornowski**

### CODE REVIEW

**Comment by sebbrudzinski** [comment]
test this psql

**Comment by sebbrudzinski** [comment]
Fix the PostgreSQL migration sql

**Comment by kmadej** [comment]
@pkornowski please try to find another solution, use IF EXISTS and maybe UPDATE IGNORE will be needed.

**Comment by pkornowski** [comment]
test this psql

**Comment by pkornowski** [comment]
test this please

**Comment by pkornowski** [comment]
test this psql

**Comment by kmadej** [comment]
you should not hardcoded the database names, please add this migration in tasks module. The migration should execute after task module starts

**Comment by kmadej** [comment]
test this please

File: modules/tasks/tasks/src/main/resources/db/migration/mysql/V1__MOTECH-2578.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- Changing serviceInterface path from org.motechproject.openmrs19.tasks.OpenMRSActionProxyService to org.motechproject.openmrs.tasks.OpenMRSActionProxyService
+
+Update motechdata.MOTECH_TASKS_ACTIONEVENT set serviceInterface='org.motechproject.openmrs.tasks.OpenMRSActionProxyService' where (serviceInterface='org.motechproject.openmrs19.tasks.OpenMRSActionProxyService' AND id <> 0);
```
Comment (kmadej) [inline modules/tasks/tasks/src/main/resources/db/migration/mysql/V1__MOTECH-2578.sql]: do not use hardcoded database name

Comment (kmadej) [inline modules/tasks/tasks/src/main/resources/db/migration/mysql/V1__MOTECH-2578.sql]: do we need this 'id <>0' ?

**Comment by kmadej** [comment]
test this psql

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this psql

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this psql

**Comment by sebbrudzinski** [comment]
test this psql

**Comment by kmadej** [comment]
test this psql

**Comment by kmadej** [comment]
@pgesek Looks good to me. ITs passed also on postgres, however I suppose there is some connection problem between github and jenkins, that's why the postgres check is not visible in this PR.

**Pull request merged**